### DISCUSSION TITLE
Getting error after completion of 1st epoch

### DISCUSSION THREAD

**Discussion by Abhishek-Prajapat** [Lightning-AI/pytorch-lightning#8339] [code help: CV] [answered]
I am training an image classifier on tpu and am getting errors after execution of the first epoch.

https://colab.research.google.com/drive/1Lgz0mF6UiLirsDltPQH5HtctmVvb7gHm?usp=sharing

**Comment by carmocca** [comment, marked as answer]
I don't see an error in the provided link.

Is this still relevant?

**Comment by Abhishek-Prajapat** [comment]
No. I think I change the notebook. I don't even remember what was the error.
